src/util/ChartUtils.ts

```diff
@@ -98,20 +98,15 @@ export const calculateActiveTickIndex = (
   axis?: BaseAxisProps,
 ) => {
   let index = -1;
-  const len = ticks?.length ?? 0;
-
-  // if there are 1 or less ticks ticks then the active tick is at index 0
-  if (len <= 1) {
-    return 0;
-  }
 
   if (axis && axis.axisType === 'angleAxis' && Math.abs(Math.abs(axis.range[1] - axis.range[0]) - 360) <= 1e-6) {
```

Comment:
It for the angleAxis, so i isolated it to make it easier to split the code later on